base_classes/NXattenuator.nxdl.xml

```diff
@@ -43,9 +43,27 @@
         TODO  SHADOW: http://ftp.esrf.eu/pub/scisoft/shadow       or (older)   http://www.nanotech.wisc.edu/shadow  
     -->
 
-    <field name="distance" type="NX_FLOAT" units="NX_LENGTH">
+    <field name="distance" type="NX_FLOAT" units="NX_LENGTH" deprecated="Use transformations/NXtransformations/distance instead">
```

Comment:
The suggested form is ambiguous with those slashes. Maybe use `transformations:NXtransformations` to indicate that the group's name and NeXus class.